bowling-challenge-1st-attempt/bowling.html

```diff
@@ -0,0 +1,172 @@
+<html>
+<head>
+<title>Bowling Scorecard App</title>
+</head>
+<body>
+  <h1>Lets go Bowling!</h1>
+  <form id="submit-roll">
+    <input id="pins-knocked-down" type="text" placeholder="Enter a pins knocked down"></input>
+    <input class='button' type="submit"></input>
+  </form>
+  <table>
+    <tr>
+      <th>Frame</th>
+      <th>Roll</th>
+      <th>Pins Knocked Down</th>
+      <th>Total Score</th>
+      <th>Bonuses</th>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>1</td>
+      <td id='pins-1-1'></td>
+      <td id='score-1-1'></td>
+      <td id='bonus-1-1'></td>
+    </tr>
+    <tr>
+      <td>1</td>
```

Comment:
hopefully a more efficient way than each pin regularly that can be found in the future